CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 All notable changes to `WP Curate` will be documented in this file.
 
+## 2.5.0 - 2024-09-17
```

Comment:
I feel like this could be 2.4.1. Installing it won't break anything.

Reply:
Updated in [0140c29](https://github.com/alleyinteractive/wp-curate/pull/235/commits/0140c296932d93cf1309a469b72f499ae5ff8b89)